terraform/pas_terraform/modules/aks/main.tf

```diff
@@ -0,0 +1,116 @@
+resource "rafay_aks_cluster" "cluster" {
```

Comment:
We should remove aks module and keep only aks_v3 or update the existing aks module to use v3 resource.

I suggest we just keep aks module and update the resource definition

Reply:
Make sense, will get this changed

Reply:
I've updated to use aks and removed aks_v3

Reply:
Done